src/include/wren.h

```diff
@@ -470,6 +474,14 @@ void wrenGetListElement(WrenVM* vm, int listSlot, int index, int elementSlot);
 // an element, use `-1` for the index.
 void wrenInsertInList(WrenVM* vm, int listSlot, int index, int elementSlot);
 
+// Retrieves a value with the key in [keySlot] from the map in [mapSlot] and stores it in
+// [valueSlot].
+void wrenGetMapValue(WrenVM* vm, int mapSlot, int keySlot, int valueSlot);
+
+// Takes the value stored at [valueSlot] and inserts it into the map stored
+// at [mapSlot] with key [keySlot].
+void wrenInsertInMap(WrenVM* vm, int mapSlot, int keySlot, int valueSlot);
```

Comment:
Opinions may differ, but why not just wrenSetMapValue?

Reply:
I was following the example set by the List api, but both are good.

Reply:
There's a lot of API-naming stuff which is difficult to answer. Whose job is it really to make that kind of decision?

Reply:
We can all make suggestions, but the decision would currently fall upon @underscorediscovery.

Reply:
My impression is that `wrenSetMapValue` is more consistent with `wrenGetMapValue` and less surprising than `insert` when looking for the inverse operation.

Reply:
I'll change the function name to match that opinion :)